CHANGELOG.md

```diff
@@ -153,6 +153,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 - Added `MovableObject` Lua function `EnableOrDisableAllScripts` that allows you to enable or disable all scripts on a `MovableObject` based on the passed in value.
 
+- New `Settings.ini` properties `MuteMaster = 0/1`, `MuteMusic = 0/1` and `MuteSound = 0/1` to control muting of master/music/sound channels without changing the volume property values.	
+
+- New `Settings.ini` property `TwoPlayerSplitscreenVertSplit = 0/1` to force two player splitscreen into a vertical split mode (horizontal by default).
+
+- Ingame settings menu was reworked to make it less useless.
```

Comment:
How is adding a feature that did not exist doesn't qualify for the `Added` section? lel
I mean technically you're right, it's a change, but then again literally everything else in the changelong can go into that section.

Reply:
Hmmm, you're probably right for controller hotswapping. I guess I was looking at it as changing controller handling so it supports hot swapping, but I guess it's really more of an add than a change.
Reworking the settings menu, if you wanna keep it general, is definitely a change rather than an add though. If you wanna go into details "Added whatever thingy to settings menu" then I guess it'd make some sense here, but I think just having a change entry for it in general is reasonable.